webapp/app.py

```diff
@@ -140,7 +140,6 @@
     subscription_centre,
     thank_you,
     unlisted_engage_page,
-    build_engage_pages_sitemap,
```

Comment:
Why is this removed?

Reply:
![image](https://github.com/canonical/ubuntu.com/assets/58276363/1181736f-41c3-4fd2-89e4-391b13f7acea)
duplicated in merge